packages/twenty-front/src/modules/ui/input/components/internal/date/components/InternalDatePicker.tsx

```diff
@@ -231,14 +231,17 @@ const StyledButtonContainer = styled.div`
   align-self: stretch;
   display: flex;
   gap: ${({ theme }) => theme.spacing(2)};
-  padding-left: ${({ theme }) => theme.spacing(2)};
+  padding: 0 ${({ theme }) => theme.spacing(2)};
 
   & .menu-item {
     height: ${({ theme }) => theme.spacing(8)};
     margin-bottom: ${({ theme }) => theme.spacing(1)};
     margin-top: ${({ theme }) => theme.spacing(1)};
     padding: 0 ${({ theme }) => theme.spacing(2)};
-    width: fit-content;
+
+    & > div {
```

Comment:
We avoid styling children through CSS.
Instead you can transform the div itself to a Styled component